queries/javascript/locals.scm

```diff
@@ -6,6 +6,10 @@
 (arrow_function) @scope
 (function_declaration) @scope
 (method_definition) @scope
+(if_statement) @scope
+(for_statement) @scope
+(switch_statement) @scope
+(try_statement) @scope
```

Comment:
try_statement doesn't actually create a new scope (it's already covered by statement_block), however `catch_clause` does. I would just change this to `catch_clause`